specification/synapse/resource-manager/Microsoft.Synapse/preview/2020-04-01-preview/sqlDatabase.json

```diff
@@ -0,0 +1,299 @@
+          },
+          "202": {
```

Comment:
202 should not be used for async PUT.  See https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creatingupdating-using-put

Reply:
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#202-accepted-and-location-headers

in our scenario if the PUT fails, we actually drop the resource. this is in line with what Microsoft.Sql RP does as well so after talking to them, they suggested to keep this 202 pattern as is.